Twilio/Rest/Chat/V1/Service/Channel/MemberList.php

```diff
@@ -106,21 +108,24 @@ public function stream($limit = null, $pageSize = null) {
      *                        efficient page size, i.e. min(limit, 1000)
      * @return MemberInstance[] Array of results
      */
-    public function read($limit = null, $pageSize = Values::NONE) {
-        return iterator_to_array($this->stream($limit, $pageSize), false);
+    public function read($options = array(), $limit = null, $pageSize = null) {
```

Comment:
Probably from adding optional params on read to channel members when there were none before.